ethereum/evmtool/src/test/resources/org/hyperledger/besu/evmtool/trace/create-eof-error.json

```diff
@@ -11,5 +11,5 @@
     "CancunEOF"
   ],
   "stdin": "",
-  "stdout": "EOF Code Invalid : STOP is only a valid opcode in containers used for runtime operations.\n"
+  "stdout": "EOF Code Invalid : incompatible_container_kind opode STOP is only valid for runtime.\n"
```

Comment:
```suggestion
  "stdout": "EOF Code Invalid : incompatible_container_kind opcode STOP is only valid for runtime.\n"
```

Reply:
Two tpyos.   One in the test case (this) and one in the code generating it (CodeV1Validation)